Toolset/resources/supporting_files/property_definitions/propertyInfo.tsv

```diff
@@ -165,6 +165,7 @@ showLines	Text baselines	Table	com.livecode.pi.boolean	true	false
 showName	Show name	Basic	com.livecode.pi.boolean	true	false			
 showSelection	Hilite selection	Basic	com.livecode.pi.boolean	true	false			
 showValue	Show value	Basic	com.livecode.pi.boolean	true	false		false			
+size	Size	Basic	com.livecode.pi.integer	true	true
```

Comment:
@livecodepanos I'm not sure of the value of this on the inspector particularly given it's empty for referenced images. If it is going to be there I suggest making the label `Size in bytes` to help avoid confusion with size as width, height.